src/epstats/toolkit/statistics.py

```diff
@@ -278,10 +280,15 @@ def required_sample_size_per_variant(
         if minimum_effect < 0:
             raise ValueError("minimum_effect must be greater than zero.")
 
+        if n_variants < 2:
+            raise ValueError("There must be at least two variants.")
+
         two_vars = 2 * (std ** 2) if std_2 is None else (std ** 2 + std_2 ** 2)
         delta = mean * minimum_effect
 
         alpha = 1 - confidence_level
+        m = n_variants - 1
+        alpha = alpha / m  # Bonferroni correction
```

Comment:
I'm not sure if it is relevant for pre-test calculation that we use Holm-Bonferroni instead of Bonferroni.

Reply:
It is not really applicable here. Holm-Bonferroni sorts the variants by their $p$-values and assigns the biggest correction to the variant with the lowest $p$-value and son. Since in the pre-test calculation we don't know the $p$-values, we assign the same correction $alpha/m$ to each variant.